flow/packet-headers/rsvp.yaml

```diff
@@ -0,0 +1,1222 @@
+            Among Those we are presently supporting "path"(value: 1) message type.
+          $ref: '#/components/schemas/Flow.RSVP.Message'
+          x-field-uid: 7
+    Flow.RSVP.Flag:
+      type: object
+      properties:
+        choice:
+            type: string
+            default: not_refresh_reduction_capable
+            x-enum:
+              not_refresh_reduction_capable:
+                x-field-uid: 1
+              refresh_reduction_capable:
+                x-field-uid: 2
+            x-field-uid: 1
+    Flow.RSVP.Length:
+      type: object
+      properties:
+        choice:
+          description: auto or configured value.
+          type: string
+          default: auto
+          x-field-uid: 1
+          x-enum:
+            auto:
+              x-field-uid: 1
+            value:
+              x-field-uid: 2
+        auto:
+          description: >-
+            The OTG implementation can provide a system generated value for this property.
+            If the OTG is unable to generate a value the default value must be used.
+          type: integer
+          format: uint32
+          maximum: 65535
+          default: 0
+          x-field-uid: 2
+        value:
+          type: integer
+          format: uint32
+          maximum: 65535
+          default: 0
+          x-field-uid: 3     
+    Flow.RSVP.Message:
+      type: object
+      properties:
+        choice:
+          type: string
+          default: path
+          x-enum:
+            path:
+              x-field-uid: 1
+          x-field-uid: 1
+        path:
+          $ref: '#/components/schemas/Flow.RSVP.PathMessage'
+          x-field-uid: 2
+    Flow.RSVP.PathMessage:
+      description: >-
+        "path" message required the follwoing list of objects in order as defined in https://www.rfc-editor.org/rfc/rfc3209.html#page-15:
+        1. SESSION
+        2. RSVP_HOP
+        3. TIME_VALUES
+        4. EXPLICIT_ROUTE [optional]
+        5. LABEL_REQUEST
+        6. SESSION_ATTRIBUTE [optional]
+        7. SENDER_TEMPLATE
+        8. SENDER_TSPEC
+        9. RECORD_ROUTE [optional]
+      type: object
+      properties:
+        objects:
+          description: >-
+            "path" message required atleast SESSION, RSVP_HOP, TIME_VALUES, LABEL_REQUEST, SENDER_TEMPLATE and SENDER_TSPEC objects in order.
+          type: array
+          minItems: 6
+          items:
+            $ref: '#/components/schemas/Flow.RSVP.PathObjects'
+          x-field-uid: 1
+    Flow.RSVP.PathObjects:
+      description: >-
+        Every RSVP object encapsulated in an RSVP message consists of a 32-bit word header and the object's contents.
+      type: object
+      properties:
+        length:
+          description: >-
+            A 16-bit field containing the total object length in bytes. 
+            Must always be a multiple of 4, and at least 4.
```

Comment:
Must always be a multiple of 4, and at least 4. -> Must always be a multiple of 4 or at least 4.

Reply:
Done.